modules/core/src/v2/coins/sol.ts

```diff
@@ -23,7 +23,7 @@ import { MethodNotImplementedError } from '../../errors';
 import base58 = require('bs58');
 
 export interface TransactionFee {
-  fee: string;
+  lamportsPerSignature: string;
```

Comment:
Here on transactionBuilder from account-lib you are setting the fee as lamportPerSignature, so if this is gonna be exposed by the core I think fee name must not changed. In order to have a common interface like in the txBuilder.

Reply:
good to know, renamed back to fee